README.md

```diff
@@ -1,208 +1,262 @@
+This is a list of microflows that can be called from within your application. They are located in the `_USE ME/Microflows` folder.
 
+|Microflow name         |Description                                                    |
+|-----------------------|---------------------------------------------------------------|
+|MessageFactory         |A microflow to create a `Message` object                       |
+|SendMessageImmediately |A microflow to send a message to GCM/APNs immediately          |
+|SendMessageQueued      |A microflow to send a message to GCM/APNs using a queue        |
+|SendMessagesImmediately|A microflow to send multiple messages to GCM/APNs immediately  |
+|SendMessagesQueued     |A microflow to send multiple messages to GCM/APNs using a queue|
 
-## Setting up Windows
-Windows requires no additional configuration for push notifications to work. Simply load the application onto a Windows 8 Phone and login to the application. The user's Windows Phone credentials should appear within the device menu.
-The notifications for Windows work using a web service, the widget within your application registers the user's unique URL. With the URL the Mendix application can pass a message to this URL.  
+Note: for sending message(s) queue to work properly, you need to ensure that the scheduled event `PushQueue` is active. In a sandbox/free app this does not require any extra step, but in a TAP environments you will need to explicitly set the scheduled event to be activated.
 
+## Setting up Apple Push Notification Server
 
-## Installing a Windows App
-In order to test and publish your Windows 8 applications you will need a Windows developer account that can be obtained from: 
+In order to proceed you need an Apple developer license and a device running Mac OS X.
 
-[Windows Dev Center](https://dev.windows.com/en-us) 
+We assume that you already have the app signing key with provisioning profile and can freely build and install your mobile app (if not, please refer to [this howto] (https://world.mendix.com/display/howto50/Publishing+a+Mendix+Hybrid+Mobile+App+in+Mobile+App+Stores#PublishingaMendixHybridMobileAppinMobileAppStores-3.SettingupAppSigningKeys)). Take into account that your App ID should use Explicit App ID and have Push Notification turned on so you could receive push notifications with your app.
 
-Upon doing so you will receive a Publisher GUID (this can be located under Dashboard > Windows Phone Store > Account). This GUID is needed by PhoneGap in order to sign your applications. 
+<img src="assets/images/apns/AppID.png"/>
 
-You can download the SDK as well as development tools from [here](https://dev.windows.com/en-us/develop/download-phone-sdk)
+If this is not the case, you need to create new App ID with Explicit App ID and Push Notification turned on. After that, you'll need to receive the new provisioning profile with this App ID and rebuild a mobile app.
```

Comment:
Push Notification**s**

Suggestion:
'download the new provisioning profile for this App ID and use it to rebuild the mobile app.'